iso15118/secc/states/iso15118_2_states.py

```diff
@@ -586,6 +586,11 @@ async def process_message(
             msg.body.certificate_installation_req
         )
 
+        # For the CertificateInstallation, the min the EVSE can do is
```

Comment:
Let's replace "EVSE" with "SECC" to be a bit more precise. (As the EVSE is just the power delivery unit, not the communication controller)